contrib/guix/manifest.scm

```diff
@@ -612,5 +612,5 @@ parse, modify and abstract ELF, PE and MachO formats.")
           ((string-contains target "-linux-")
            (list (make-bitcoin-cross-toolchain target)))
           ((string-contains target "darwin")
-           (list clang-toolchain-10 binutils cmake-minimal xorriso python-signapple))
+           (list clang-toolchain-15 binutils cmake-minimal xorriso python-signapple))
```

Comment:
changes in `ci/test/00_setup_env_mac.sh` is missing:

```
-export PACKAGES="cmake libz-dev libtinfo5 python3-setuptools xorriso"
+export PACKAGES="cmake libz-dev python3-setuptools xorriso"
```

due to missing backport bitcoin#21112

Either mark #28337 partial or do 21112 beforehead